configuration/ldap/ldap_config.py

```diff
@@ -86,12 +86,22 @@ def _import_group_type(group_type_name):
 # Define a group required to login.
 AUTH_LDAP_REQUIRE_GROUP = environ.get('AUTH_LDAP_REQUIRE_GROUP_DN')
 
+# If non-empty string, AUTH_LDAP_REQUIRE_GROUP will be treated as a list delimited by this separator
+AUTH_LDAP_REQUIRE_GROUP_SEPARATOR = environ.get('AUTH_LDAP_REQUIRE_GROUP_DN_SEPARATOR', '')
+
```

Comment:
TBH, I'm not thrilled about this change. If the variable `AUTH_LDAP_REQUIRE_GROUP_DN` can be a list anyway, why not treat is as a list all the time and use a fixed separator, instead of a configurable one? Also, properly trimming each group.

Additionally, the default value should probably be `None` instead of `''`?

```suggestion
# If non-empty string, AUTH_LDAP_REQUIRE_GROUP will be treated as a list delimited by this separator
AUTH_LDAP_REQUIRE_GROUP_SEPARATOR = environ.get('AUTH_LDAP_REQUIRE_GROUP_DN_SEPARATOR', None)

```

Reply:
Or we could revert it for now. Probably better then an untested quick fix.